cmd/cql/internal/web.go

```diff
@@ -0,0 +1,63 @@
+package internal
+
+import (
+	"net/http"
+
+	"github.com/CovenantSQL/CovenantSQL/sqlchain/observer"
+	"github.com/CovenantSQL/CovenantSQL/utils"
+)
+
+var (
+	webAddr string // Web addr
+
+	webService    *observer.Service
+	webHTTPServer *http.Server
+)
+
+var CmdWeb = &Command{
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var CmdWeb should have comment or be unexported